tests/spread/testcraft/test-cmd/spread.yaml

```diff
@@ -0,0 +1,13 @@
+backends:
+  craft:
+    type: adhoc
+    allocate: "false"
+    systems:
+      - ubuntu-24.04-64:
```

Comment:
from looking at the failures, possibly this needs to be `ubuntu-24.04`

Reply:
```suggestion
      - ubuntu-24.04:
```